src/shared/components/oncoprint/DataUtils.spec.ts

```diff
@@ -1287,6 +1288,24 @@ describe("DataUtils", ()=>{
                {geneset_id:"MY_FAVORITE_GENE_SET-3", study_id:"study", profile_data:7}
            );
        });
+
+       it('adds thresholdType and category to trackDatum', () => {
+           let data = [
+            {value:8, thresholdType: '>' as '>'},
+           ];
+           const partialTrackDatum = {};
+           fillHeatmapTrackDatum<ITreatmentHeatmapTrackDatum, "treatment_id">(
+            partialTrackDatum,
+            "treatment_id",
+            "TREATMENT_ID_1",
+            {patientId:"patient", studyId:"study"} as Sample,
+            data
+           )
+           assert.deepEqual(
+            partialTrackDatum,
+            {treatment_id: "TREATMENT_ID_1", study_id:"study", profile_data:-10, thresholdType: '>', category: '>8.00'}
+           );
```

Comment:
can you more exhaustively test this, with different threshold types and edge cases eg value right on the threshold, different sort orders

Reply:
Added tons of test testing all kinds of edge cases. Also updated DataUtils to meet test requirements ;).

Reply:
great, thank you